miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -462,20 +462,16 @@ export.get_block_number
     # => [num]
 end
 
-#! Returns the block hash of the last known block at the time of transaction execution.
```

Comment:
Changed from "last known" to "reference block" because a user may intentionally use an older block.